build.sbt

```diff
@@ -166,14 +166,15 @@ ThisBuild / githubWorkflowPublishPreamble +=
   WorkflowStep.Use(
     UseRef.Public("actions", "setup-node", "v4"),
     Map(
-      "node-version" -> "20",
+      "node-version" -> "22",
       "registry-url" -> "https://registry.npmjs.org"
     )
   )
 
 ThisBuild / githubWorkflowPublish ++= Seq(
   WorkflowStep.Sbt(List("npm/npmPublish"),
                    name = Some("NPM Publish"),
-                   env = Map("NODE_AUTH_TOKEN" -> s"$${{ secrets.NPM_REPO_TOKEN }}")
+                   env = Map("NODE_AUTH_TOKEN" -> s"$${{ secrets.NPM_REPO_TOKEN }}"),
+                   cond = Some("startsWith(github.ref, 'refs/tags/v'")
```

Comment:
Is it missing a closing paren here?

Reply:
Are you sure? It seems to be one short in ci.yml

Reply:
Ohhh in the workflow! Yes there's a paren missing there, good eye. I was just trying to figure out why CI didn't run 😅